ferolino_jasmin/javascript.js

```diff
@@ -79,4 +79,49 @@ function renderComments() {
 }
 
 sortDropdown.addEventListener("change", renderComments);
-renderComments()
\ No newline at end of file
+renderComments()
+
+async function searchCountry() {
+  const countryInput = document.getElementById('countryInput').value;
```

Comment:
### IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

This also applies to lines 95 and 113 in your JS file.